erpnext/stock/doctype/pick_list_item/pick_list_item.json

```diff
@@ -200,10 +202,21 @@
    "label": "COA Batch",
    "options": "Batch",
    "read_only": 1
+  },
+  {
+   "fieldname": "cb_reference",
+   "fieldtype": "Column Break",
+   "label": "Pick List Item-cb_reference"
+  },
+  {
+   "fieldname": "so_customer",
+   "fieldtype": "Data",
+   "label": "SO Customer",
+   "read_only": 1
```

Comment:
where is `in_list_view`, `fetch_from`? Check the bloomstack_core file properly!!

Reply:
Hi Sahil,
I have made the suggested changes and raised the pr.

PR link: https://github.com/Bloomstack/erpnext/pull/1590